common/service/dynamicconfig/constants.go

```diff
@@ -904,6 +905,9 @@ const (
 
 	// lastKeyForTest must be the last one in this const group for testing purpose
 	lastKeyForTest
+
+	// EnableRequestActivityLocalDispatchByDomain indicates if client can request to start activity locally
```

Comment:
// EnableActivityLocalDispatchByDomain allows worker to dispatch activity tasks through local tunnel after decisions are made. This is an performance optimization to skip activity scheduling efforts.